app/views/books/index.html.erb

```diff
@@ -1,24 +1,31 @@
-<h1>Bookstore</h1>
 <p>
-  <%= form_tag(root_path, method: :get) do %>
-    <%= text_field_tag :search %>
-    <%= submit_tag "Search" %>
+  <%= form_tag books_path, method: :get, class: 'form-inline' do %>
+    <div class='form-group'>
+      <%= text_field_tag :search, params[:search], class: 'form-control' %>
+    </div>
+    <%= submit_tag "Search", class: 'btn btn-default' %>
   <% end %>
-<table>
+</p>
+<%= link_to "Sort by Most Popular", root_path(sort: "times_purchased") %></p>
+<table class='table'>
   <thead>
     <tr>
       <th>Title</th>
       <th>Author</th>
       <th>Published Date</th>
       <th>Price</th>
+      <th>Times Purchased</th>
     </tr>
   <% @books.each do |book| %>
     <tr>
       <td><%= book.title %></td>
       <td><%= book.author %></td>
       <td><%= book.published_date %></td>
-      <td><%= book.price_cents %></td>
+      <td><%= book.price.format %></td>
+      <td><%= book.times_purchased %></td>
```

Comment:
While @jahammo2 is correct, we are using bootstrap in this app so we are not concerning ourselves with styles. A table is an acceptable choice here.

However, I would like to note that I do not like the use of a `times_purchased` here. Instead of caching some arbitrary purchase count, just do a `book.line_items.count`. That way, you do not have to trouble yourself with the difficulties of keeping this counter attribute up to date.